software/globaleaks.yml

```
@@ -0,0 +1,13 @@
+name: "GlobaLeaks Whistleblowing Software"
+website_url: "https://www.globaleaks.org/"
+source_code_url: "https://github.com/globaleaks/whistleblowing-software"
+demo_url: "https://demo.globaleaks.org"
+description: "A whistleblowing software enabling anyone to easily set up and maintain a secure reporting platform. "
```

```suggestion
description: "Whistleblowing software enabling anyone to easily set up and maintain a secure reporting platform. "
```

https://english.stackexchange.com/questions/12990/why-dont-we-use-the-indefinite-article-with-software